pkg/cmd/clustertask/describe_test.go

```diff
@@ -271,6 +271,31 @@ func Test_ClusterTaskDescribe(t *testing.T) {
 	}
 }
 
+func TestClusterTaskDescribe_WithoutNameIfOnlyOneClusterTaskPresent(t *testing.T) {
+	cstasks := []*v1alpha1.ClusterTask{
+		tb.ClusterTask("task-1"),
```

Comment:
Please try to avoid the use of `tb` or test builders when writing unit tests as they will be deprecated in the future. You can use structs instead like you did in the `v1beta1` test.